packages/pwa-kit-dev/src/ssr/server/build-dev-server.js

```diff
@@ -143,6 +143,18 @@ export const DevServerMixin = {
                 dotFiles: 'deny'
             })
         )
+
+        app.get('/__mrt/clear-browser-data', (_, res) => {
+            console.log(
+                chalk.cyan('Clearing browser data'),
+                '(cache, service worker, web storage for browsers supporting Clear-Site-Data header)'
+            )
```

Comment:
I wanted to make our clearing-browser-data more visible to the other developers.